apps/supaclk/app.js

```diff
@@ -0,0 +1,195 @@
+    32,
+    atob("BAUHCwsNEQQGBgkKBQkFBwwHCgkKCQoJCgoFBQsLCwkQDAwNDgwLDQ8HBw0LEQ0PDA8NCwwMDBINCwsGBwY="),
+    18|65536
+  );
+}
+
+// Actually draw the watch face
+let draw = function() {
+  g.reset().setColor(g.theme.fg).setBgColor(g.theme.bg);
+  g.clearRect(R.x,R.y, R.x2, R.y2 - 60);
+  var date = new Date();
+  let minutes = date.getMinutes();
+  minutes = (minutes < 10 ? '0' : '') + minutes;
+
+  g.setColor(g.theme.fg)
+  // Time
+  const yt =  R.y + 92 - 20 - 30 + 6 + 10;
+  const xt = R.w/2 - 5;
+  let hours = date.getHours()+'';
+  g.setFontAlign(1, 0).setFontPlayfairDisplay().drawString(hours, xt - 8, yt);
+  g.setFontAlign(0, 0).setFontPlayfairDisplay().drawString(':', xt, yt);
+  g.setFontAlign(-1, 0).setFontPlayfairDisplay().drawString(minutes, xt + 8, yt);
+  // logo
+  g.drawImage(supaClockImg, R.x2 - supaClockImg.width - 2, R.y + 2);
+  // dow + date
+  let dateStr = require("locale").dow(date).toUpperCase() + '\n' + require("locale").month(date, 2).toUpperCase() + ' ' + date.getDate();
+  g.setFont('6x8').setFontAlign(-1, 0).drawString(dateStr, R.x2 - supaClockImg.width - 2, R.y + 42 - 30 + 8);
+
+  if (drawTimeout) clearTimeout(drawTimeout);
+  drawTimeout = setTimeout(function() {
+    drawTimeout = undefined;
+    draw();
+  }, 60000 - (Date.now() % 60000));
+};
+
+let clockInfoDraw = (itm, info, options) => {
+  g.reset().setFontPlayfairDisplaySm().setColor(g.theme.fg).setBgColor(g.theme.bg);
+  if (options.focus) g.setBgColor("#FF0");
+  g.clearRect({x:options.x,y:options.y,w:options.w,h:options.h});
+
+  if (info.img) {
+    g.drawImage(info.img, options.x+1,options.y+2);
+  }
+  var text = info.text.toString().toUpperCase();
+  if (g.setFontPlayfairDisplaySm().stringWidth(text)+24-2>options.w) g.setFont("8x12");
+  g.setFontAlign(-1,-1).drawString(text, options.x+24+3, options.y+6);
+};
+
+let clockInfoDrawR = (itm, info, options) => {
+  g.reset().setFontPlayfairDisplaySm().setColor(g.theme.fg).setBgColor(g.theme.bg);
+  if (options.focus) g.setBgColor("#FF0");
+  g.clearRect({x:options.x,y:options.y,w:options.w,h:options.h});
+
+  if (info.img) {
+    g.drawImage(info.img, options.x + options.w-1-24,options.y+2);
+  }
+  var text = info.text.toString().toUpperCase();
+  if (g.setFontPlayfairDisplaySm().stringWidth(text)+24-2>options.w) g.setFont("8x12");
```

Comment:
... also here `"8x12"` is not available to `setFont` as per review comment above.